tsl/test/expected/bgw_custom.out

```diff
@@ -108,6 +119,12 @@ SELECT * FROM _timescaledb_config.bgw_job WHERE id >= 1000;
 ----+------------------+-------------------+-------------+-------------+--------------+-------------+-----------+-------+-----------+---------------+--------
 (0 rows)
 
+SELECT count(*) FROM timescaledb_information.jobs WHERE job_id >= 1000;
```

Comment:
Isn't it better just replace the internal table with the view in the above select?

Reply:
BUMP